src/main.rs

```diff
@@ -336,6 +336,10 @@ impl From<CheckRelease> for cargo_semver_checks::Check {
             check.with_heuristically_included_features();
         }
         let mut mutual_features = value.features;
+        // Treat --features="" as a no-op like cargo does
+        if mutual_features == [""] {
+            mutual_features.clear();
+        }
```

Comment:
Two questions:
- Does this also work in the case of `--features '' --features xyz`? This flag can be repeated and we presumably want the empty-string feature to always be a no-op.
- Should we also make this same behavior apply to `--current-features` and `--baseline-features` as well? Those flags are very similar (`--features` is shorthand for using both of those flags at once with the same inputs) so it feels like they should behave similarly in this aspect as well.

Reply:
I didn't know you could specify features multiple times in an additive fashion. In this case, your suggestion does indeed make sense.

Reply:
Changed in the latest commit.